### PR TITLE
Get all property sources for health indicator

### LINKED ISSUE #1806: Actuator health check missing details in 3.0.x
**Describe the bug**
Hi, using cloud config client version 3.0.0 and native profile for configuration I always get that configuration property sources could not be loaded and status is UNKNOWN when calling the actuator health check.

```
"clientConfigServer": {
  "status": "UNKNOWN",
  "details": {
    "error": "no property sources located"
  }
}
```

Comparing to spring boot 1.x I was receiving:

```
"configServer": {
  "status": "UP",
  "propertySources": [
     "file:C:\\Users\user\projects\tagging.yml",
  ]
}
```

Here is my configuration:

```
spring:
  application:
    name: my-app
  cloud:
    config:
      discovery:
        enabled: true
        serviceId: config-server
      failFast: false
      retry:
       initialInterval: 10000
       maxInterval: 60000
       maxAttempts: 12
       multiplier: 10.1
```
I see also the log that probably should be "CompositePropertySource" but is Bootstrap one:

`2021-02-23 10:11:15.402  INFO b.c.PropertySourceBootstrapConfiguration : Located property source: [BootstrapPropertySource {name='bootstrapProperties-configClient'}, 
BootstrapPropertySource {name='bootstrapProperties-file:C:\Users\user\properties.yml'}]`

### PR DESCRIPTION
**Pull request by ryanjbaxter**
Fixes #1806

### CODE REVIEW

File: spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java [hunk 1]
```diff
@@ -57,6 +57,11 @@
 
 public class ConfigServerConfigDataLoader implements ConfigDataLoader<ConfigServerConfigDataResource>, Ordered {
 
+	/**
+	 * PropertySource name for the config client.
+	 */
+	public static final String CONFIG_CLIENT_PROPERTYSOURCE_NAME = "configClient";
```
Comment (spencergibb) [inline spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java:63]: Move to health indicator?

**Review by spencergibb — APPROVED** [review]
LGTM other than the location of a constant.

File: spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java (hunk 1 above)
Comment (ryanjbaxter) [reply spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java:63]: 🤦‍♂️ I meant to actually use that variable in that class, it looks like that is where we are creating the PropertySource, I just pushed the change

Comment (spencergibb) [reply spring-cloud-config-client/src/main/java/org/springframework/cloud/config/client/ConfigServerConfigDataLoader.java:63]: 👍🏻

**Review by spencergibb — APPROVED** [review]

**Pull request merged**